bbcode-src/preset.js

```diff
@@ -26,6 +27,12 @@ const tags = {
   color,
   divide,
   font,
+  h,
```

Comment:
@kyuukestu  For backwards compatibility sake, we need to add the [sh] tag to this list as well. It should map to the `<h2>` bbcode logic as the translation is the same. For consistency, I think we should add [h1] as well, which will map to the same logic as the [h] tag. 

In the end we'll have backwards compat [h] and [sh] tags and the new/preferred method [h1-6] tags.

Reply:
Gotcha, I'll get right on that o7